challenges/stats_preview_card_component/style.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Outfit:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lexend+Deca&family=Outfit:wght@400;700&display=swap');
```

Comment:
Is this font used anywhere?

Reply:
They are both used (Inter and Lexend Deca) in the text. Inter for font-bold text and L-D for font-normal text

Reply:
Ok. You can get multiple fonts using one URL to reduce number of HTTP requests.